### ISSUE TITLE
QCArchive Integration

### ISSUE THREAD

**Issue by dgasmith** [obackhouse/gmtkn#1]
This is a great repo of data! As a thought MolSSI has the QCArchive (https://qcarchive.molssi.org) project to curate known datasets in the literature. Helping to curate data from non-MolSSI personnel is new to us, but we are current building the structures for contributions here: https://github.com/MolSSI/MQCAS

**Comment by obackhouse** [comment]
I'd be happy to integrate this (or have this be integrated). Also just to be crystal clear this is not my original data, I just wrote a python scraper to build the python data structures from it. The QCArchive looks great and I will certainly keep an eye on this.

**Comment by dgasmith** [comment]
Absolutely, most structured data for these datasets is not original. We have several required field to handle the citations which is quite critical in this regard. I think we may be integrating GMTKN soon, but I wanted to give you a heads up here as well if you are interested in using or donating additional structured data.

**Comment by obackhouse** [comment]
Ok, I understand - you're very welcome to use any of the contents of this repo if it is helpful to you, and I will look into contributing to the project. I didn't put much thought into the data structures here since this was mostly for personal use. 

There is also the G1 and G2 small molecule datasets which I have found very useful to my research - however, I never got hold of any published standard regarding coordinates for these molecules, I have previously optimised them myself or used CCCBDB data.

**Comment by beckhamwjc** [comment]
I just want to say thank you to @obackhouse . This is indeed beneficial for anyone who would like to compare their DFA with common DFAs. GMTKN55 is a complete and comprehensive benchmark dataset, but it's also painful to organize the whole dataset before any computation. Now with this nicely organized python implementation, we can immediately adopt PySCF or other _ab initio_ programs based on python.
Good job, and thanks again.

**Comment by obackhouse** [comment]
> I just want to say thank you to @obackhouse . This is indeed beneficial for anyone who would like to compare their DFA with common DFAs. GMTKN55 is a complete and comprehensive benchmark dataset, but it's also painful to organize the whole dataset before any computation. Now with this nicely organized python implementation, we can immediately adopt PySCF or other _ab initio_ programs based on python. Good job, and thanks again.

No problem - I'm glad to hear that you find it useful!
